src/Panel/Panel.jsx

```diff
@@ -0,0 +1,48 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes} from "react";
+
+import "./Panel.less";
+
+/*
+ * Panel is various types of panel.
+*/
+export default function Panel({children, type, title}) {
+  const {Icons, cssClass} = Panel;
+
+  return (
+    <div className={classnames(`Panel--${type}`, cssClass.CONTAINER)}>
+      <div className={cssClass.HEADER}>
+        <i className={classnames("fa", Icons[type])} />
```

Comment:
browsers build indexes of classnames to elements, and specifying elements via nesting can often be less efficient and brittle to structural changes in the HTML than if you just gave the element a CSS class itself. He's saying to keep the current nesting of the HTML, but to refer to his element in CSS by a classname.